src/Resources/skeleton/twig/Extension.tpl.php

```diff
@@ -11,7 +11,10 @@ class <?= $class_name ?> extends AbstractExtension
     public function getFilters(): array
     {
         return [
-            new TwigFilter('filter_name', [$this, 'doSomething'], ['is_safe' => ['html']]),
+            // If your filter generates SAFE HTML, you should add a third
+            // parameter: ['is_safe' => ['html']]
+            // Reference: https://twig.symfony.com/doc/2.x/advanced.html#automatic-escaping
```

Comment:
Or add an --safe-html option on the command? Such comments will be committed usually, thus be noise to many lazy fools (all of us ;) )

Reply:
Almost all code generated by this bundle should be updated by the end user. So it prefer to not add a new option
For example the callback on this next **next** line should be updated.